inspirehep/base/templates/search/results.html

```diff
@@ -148,7 +148,7 @@
     <div class="panel-body" >
       <div class="row">
         <div class="col-md-9">
-          <span class="text-left {{ request.args.get('cc')|lower() }}" id="clear-facets-message">{{ pagination.total_count }} of {{ request.args.get('p')|number_of_search_results(request.args.get('cc')) }} results match the filters you selected.</span><br/>
+          <span class="text-left {{ request.args.get('cc')|lower() }}" id="clear-facets-message">{{ pagination.total_count }} of {{ request.args.get('p', '')|number_of_search_results(collection.name) }} results match the filters you selected.</span><br/>
```

Comment:
Yes, this page is rendered by the view in `search.py` which adds `collection` to the context so it should always be there.